packages/packer-plugins/fetch-plugin-docs.ts

```diff
@@ -0,0 +1,74 @@
+import fetch from 'isomorphic-unfetch'
+import { parseSourceZip } from './parse-source-zip'
+import { parseDocsZip } from './parse-docs-zip'
+import { RawPluginFile } from './types'
+
+interface Options {
+  repo: string
+  tag: string
+}
+
+// Given a repo and tag,
+//
+// return [null, docsMdxFiles] if docs files
+// are successfully fetched and valid,
+// where docsMdxFiles is an array of { filePath, fileString } items.
+//
+// otherwise, return [err, null]
+// where err is an error message describing whether the
+// docs files were missing or invalid, with a path to resolution
+async function fetchDocsFiles({ repo, tag }: Options) {
+  // If there's a docs.zip asset, we'll prefer that
+  const docsZipUrl =
+    tag === 'latest'
+      ? `https://github.com/${repo}/releases/latest/download/docs.zip`
+      : `https://github.com/${repo}/releases/download/${tag}/docs.zip`
+  const docsZipResponse = await fetch(docsZipUrl, { method: 'GET' })
```

Comment:
Q: is there a need for authenticated requests here? (w.r.t. rate limiting)

Reply:
Good question, thus far I haven't run into any rate limiting / auth issues that I have noticed with these calls. The auth issues I think were stemming from when we were using the actual GH API to make un-authed API calls.